src/server_manager/install_scripts/install_server.sh

```diff
@@ -158,7 +158,10 @@ function fetch() {
 }
 
 function install_docker() {
+  umask 0022
+  (chmod a+r /usr/share/keyrings/docker-archive-keyring.gpg 2>/dev/null || true)
```

Comment:
Do you need the umask call since you are already updating the permission here?

Reply:
Yes, both are needed. chmod call is for already created keyring file from the previous run.